chxtools/xfuncs.py

```diff
@@ -17,13 +17,16 @@
 
 import pylab as pl
 import numpy as np
-from os import listdir
-from os.path import isfile, join
 import re
 
+from pkg_resources import resource_filename as rs_fn
+from pathlib import Path
+
 # path to X-ray data files
-datapath='/home/xf11id/Repos/chxtools/chxtools/X-ray_database/'
-default_id='CHX_IVU20_05272017';
+# This is a Path object:
+datapath = Path(rs_fn('chxtools', 'X-ray_database'))
+xdatafiles = [str(f.relative_to(datapath)) for f in datapath.glob('*') if (Path(datapath) / Path(f)).is_file()]   
```

Comment:
Nice. This is the first I'm hearing about `Path.glob`.

Reply:
Yes, the lib is amazing! I wanted to use `f.relative_to(datapath)` instead of `str(f.relative_to(datapath))`, but it appeared `re.search('(?<=mu_)\w+', xdatafiles[i])` does not support PosixPath objects.